ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -430,16 +436,19 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
       );
     }
 
+    openContactWidget() {
+      const {profileState: {profile: {contactEmail, familyName, givenName, username}}} = this.props;
+      openZendeskWidget(givenName, familyName, username, contactEmail);
+    }
+
     renderBillingDescription() {
       return <div>
         The <i>All of Us</i> Program provides $300 in free credits per user. Please refer to
         <StyledAnchorTag href={'https://aousupporthelp.zendesk.com/hc/en-us/sections' +
         '/360008099991-Questions-About-Billing'} target='_blank'> &nbsp;this article
         </StyledAnchorTag> to learn more about the free credit
         program and how it can be used. Once you have used up your free credits, you can request
-        additional credits by <StyledAnchorTag href={'https://aousupporthelp.zendesk.' +
-      'com/hc/en-us/articles/360039539411-How-to-Create-a-Billing-Account>'} target='_blank'>
-        &nbsp;contacting support</StyledAnchorTag>.
+        additional credits by <span style={styles.link} onClick={() => this.openContactWidget()}> contact us</span>.
```

Comment:
shouldn't this text be "contacting support" with no space?

Reply:
i have removed the space however span was ignoring the space hence the screen shot attached in the description remains the same